utilities/after_success.sh

```diff
@@ -32,7 +32,7 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -
 
         cd ..
         utilities/update_docs_version.sh # Update version in READMEs
-        mvn deploy -DskipTests=true --settings target/travis/settings.xml -P sign-deploy
+        mvn clean deploy --settings ~/.m2/settings.xml -P sign-deploy
     else
         mvn deploy -DskipTests=true -Dgpg.skip=true --settings target/travis/settings.xml
```

Comment:
Are we sure we want to re-compile and re-run the tests?
It seems to me that the cause of the issue is `mvn site` and we do not run it for snapshots. I would avoid making travis build even slower if not strictly necessary.